remove_unwanted_files.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+export PATH=/bin:/usr/bin
+IFS='
+'
+
+# Remove files that have something in the distrust field
+grep -l '^# distrust=.' ./*.crt | xargs rm
+
+# Remove files that don't have serverAuth in the openssl-trust field
+for cert in $(ls *.crt); do
+  if ! grep -q 'serverAuth' $cert; then
+    rm $cert
```

Comment:
It's worth quoting the bash variable so that it doesn't accidentally expand from bad inputs and remove things that we don't actually want to remove.